tests/test_library_conf.py

```diff
@@ -62,5 +62,121 @@ def test_trace_header_tags(self):
         tag_config_list = full_tag_config_list[1::]
 
         tags = {item.split(":")[1]: self.headers[item.split(":")[0]] for item in tag_config_list}
+        interfaces.library.add_span_tag_validation(request=self.r, tags=tags)
+
+
+@irrelevant(library="cpp")
+@released(dotnet="2.1.0", golang="1.53.0", java="0.102.0", nodejs="?", php="0.74.0", python="1.2.1", ruby="?")
```

Comment:
What about here and my other new scenarios @cbeauchesne ?
The other tests still have `@released(java="0.102.0", php="?", python="1.2.1")`. I imagine the changes have not been made in the manifests for Java, php and python, so those are the only languages that need to be included in the `released` decorator?

Reply:
I'm currently migrating all those declarations in manifest file, I've done C++, .NET, golang, nodejs and ruby. The last three tracer will be done in coming days :)